metabrainz/api/views/musicbrainz.py

```diff
@@ -36,39 +37,39 @@ def replication_check():
                    for e in os.listdir(current_app.config['REPLICATION_PACKETS_DIR'])]
     except OSError as e:
         logging.warning(e)
-        return Response("UNKNOWN " + str(e), mimetype='text/plain')
+        return Response(gettext("UNKNOWN " + str(e)), mimetype='text/plain')
```

Comment:
These strings don't need to be translated. This is a function for internal use. Regular users will never need to access it.